data/futures/csvconfig/instrumentconfig.csv

```diff
@@ -322,12 +327,12 @@ MSCICHINA,MSCI China Free Net Total Return (USD) Index,50,USD,Equity,0,0,0,EMEA
 MSCICHINANET,MSCI China Free Net Total Return (USD) Index (HKFE),50,USD,Equity,0,0,0,ASIA
 MSCIEAFA,MSCI EAFA Index,50,USD,Equity,0,0,0,US
 MSCIEAFA-TOT,MSCI EAFE Net Total Return Index,20,USD,Equity,0,0,0,US
-MSCIEM,MSCI EM INDEX,50,USD,Equity,0,0,0,EMEA
+MSCIEM,MSCI EM Net Total Return Index,100,USD,Equity,0,0,0,EMEA
 MSCIEM-LIFFE,MSCI EMERGING MARKETS INDEX (CBOE),50,USD,Equity,0,0,0,US
 MSCIEMASIA,MSCI Emerging Markets Asia Net Total Return,100,USD,Equity,0,0,0,US
 MSCIEURONET,MSCI Europe Net Return EUR Index,100,EUR,Equity,0,0,0,EMEA
 MSCIEURONET-ICE,The MSCI Europe Financials Net Total Return,100,EUR,Equity,0,0,0,EMEA
-MSCIEUROPE,MSCI Europe NTR,10,USD,Equity,0,0,0,EMEA
+MSCIEUROPE,MSCI Europe NTR,100,USD,Equity,0,0,0,EMEA
```

Comment:
Strangely, IB only lists a multiplier (100) for this, but not pointsize. Presuming a magnifier of 1, the pointsize would be 100. Is there some alternative way of finding out the pointsize?